riff-raff.yaml

```diff
@@ -0,0 +1,10 @@
+stacks: [targeting]
+regions: [eu-west-1]
+deployments:
+    storybook-static:
```

Comment:
It occurs to me to mention that this string `storybook-static` will be used as the `App` in RiffRaff. This is quite a general name, is there something that would better fit our application? We're using `braze-components-static` elsewhere.

Reply:
One helpful tool is the config validator in RiffRaff's UI, which shows what it understands from a RiffRaff.yaml file:

  https://riffraff.gutools.co.uk/configuration/validation

Reply:
Ah that's interesting. I'd been thinking about the app in Riff Raff as being scoped/belonging to the project. So you're right `storybook-static` is general, but scoped to `targeting:braze-components` it makes more sense. Definitely happy to change, though I'd be keen to keep `storybook` in there somewhere.

Reply:
You can set the app explicitly in the RiffRaff.yaml if you don't want it to be drawn from the deployment name. I kind of feel like the App in RiffRaff should be the same as the App we're using elsewhere, but truthfully I'm not sure it matters heaps if its inconsistent just in RiffRaff. I feel like maybe I need to talk to DevX about whether RiffRaff's understanding of the "app" is important anywhere.